src/Kestrel.Core/Adapter/Internal/RawStream.cs

```diff
@@ -69,7 +69,7 @@ public override Task<int> ReadAsync(byte[] buffer, int offset, int count, Cancel
             return ReadAsyncInternal(new Memory<byte>(buffer, offset, count)).AsTask();
         }
 
-#if NETCOREAPP2_1
+#if NETCOREAPP2_2
```

Comment:
Product assemblies should only move forward to 2.2 as needed.

Reply:
Correct, nice catch.